session.go

```diff
@@ -47,17 +47,18 @@ var defaultClientOpts = []option.ClientOption{
 const defaultPriority = pb.RequestOptions_PRIORITY_MEDIUM
 
 type Session struct {
-	projectId       string
-	instanceId      string
-	databaseId      string
-	client          *spanner.Client
-	adminClient     *adminapi.DatabaseAdminClient
-	clientConfig    spanner.ClientConfig
-	clientOpts      []option.ClientOption
-	defaultPriority pb.RequestOptions_Priority
-	directedRead    *pb.DirectedReadOptions
-	tc              *transactionContext
-	tcMutex         sync.Mutex // Guard a critical section for transaction.
+	projectId                  string
+	instanceId                 string
+	databaseId                 string
+	client                     *spanner.Client
+	adminClient                *adminapi.DatabaseAdminClient
+	clientConfig               spanner.ClientConfig
+	clientOpts                 []option.ClientOption
+	defaultPriority            pb.RequestOptions_Priority
+	directedRead               *pb.DirectedReadOptions
+	tc                         *transactionContext
+	tcMutex                    sync.Mutex // Guard a critical section for transaction.
+	protoDescriptorFileContent []byte
```

Comment:
nit: maybe we can use more concise variable name like `protoDescriptor` because the session doesn't care if the proto descriptor was coming from file content or not.

Reply:
Thank you! 
Fixed in https://github.com/cloudspannerecosystem/spanner-cli/pull/195/commits/30e2eabf6878dac74f75e77d083c42fc1d8ce8b6